components/Receiver.js

```diff
@@ -1,39 +1,52 @@
-import React, {useState} from "react";
-import { View, Text, TouchableOpacity, StyleSheet, FlatList } from "react-native";
+import React, { Component } from "react";
+import { View, Text, StyleSheet, SafeAreaView } from "react-native";
+import { Header } from 'react-native-elements';
+import { ScrollView } from "react-native-gesture-handler";
 import Checkin from './Checkin';
-import ReceiverTimes from './ReceiverTimes';
 
-const Receiver = ({history, navigation}) => {
-  const [timers, editTimers] = useState([
-    {id: 1, category: "Wake-up", time: '7:00', window: '0:15'},
-    {id: 2, category: "Take meds", time: '12:00', window: '1:00'},
-    {id: 3, category: "go to sleep", time: '22:00', window: '1:00'}
-  ])
-  return (
-    <View style={styles.careReceiver}>
-      <Text>Care Receiver</Text>
-      <TouchableOpacity style={styles.listItem} onPress={() => navigation.navigate('Home')}>
-        <View style={styles.listItemView}>
-          <Text>Back</Text>
-        </View>
-      </TouchableOpacity>
-      <Checkin />
-      <FlatList
-        data={timers}
-        renderItem={({ item }) => (
-          <ReceiverTimes item={item} />
-        )}
-        keyExtractor={(item) => item.id.toString()}
-      />
-    </View>
-  );
+class Receiver extends Component {
+  constructor() {
+    super();
+    this.state = {
+      timers: [
+        {id: 1, category: "Wake-up", time: '7:00', window: '0:15'},
+        {id: 2, category: "Take meds", time: '12:00', window: '1:00'},
+        {id: 3, category: "go to sleep", time: '22:00', window: '1:00'}
```

Comment:
I know we already talked about it but just to make a comment out of it, we'll need to add a "completed" key to these objects.